lib/electron-browser.js

```diff
@@ -5,6 +5,7 @@ var EventEmitter = require('events').EventEmitter
 var inherits = require('util').inherits
 var Split = require('split2')
 var debug = require('debug')('airtap:electron')
+var electron = require('electron')
```

Comment:
This is an (implicit) peer dependency, so it should be lazily required.

Reply:
Good catch. I clearly wasn't thinking, while thinking :smile: